MapFilterFold/MFFTest/Test1.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Functional;
+namespace MFF
```

Comment:
В .net 8 уже доступны file-scoped namespace
```suggestion
namespace MFF;
```